.github/workflows/deploy-stage.yml

```
@@ -0,0 +1,38 @@
+        branches: [staging-betterfinco]
+jobs:
+    deploy:
+        runs-on: ubuntu-latest
+        environment: staging
+        strategy:
+            matrix:
+                node-version: [20.x]
+        steps:
+            - uses: actions/checkout@master
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v1
```

actionlint (1.7.8)

the runner of "actions/setup-node@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)